cc-proxy/src/birthdayFundraiseReminder.js

```diff
@@ -0,0 +1,113 @@
+const fetch = require('node-fetch');
+const dayjs = require('dayjs');
+
+async function birthdayFundraiseReminder(req, res) {
+	const token = process.env.APP_TOKEN;
+	const today = dayjs().toISOString();
+	var nextBirthdayUpdated = [];
+	var nextBirthdayPassedUpdated = [];
+
+	// get users whose next birthday that have passed
+	const usersBirthdayPassedUrl = `https://api.raisely.com/v3/users?private=true&private.nextBirthdayLTE=${today}`;
+	const usersNextBirthdayAbsentUrl = `https://api.raisely.com/v3/users?private=1&nextBirthdayAbsent=1&dateOfBirthPresent=1`;
+
+	// request to grab users with empty next birthday data + birthday is present
+	// to update next birthday
+	const usersNextBirthdayAbsentResponse = await fetch(
+		usersNextBirthdayAbsentUrl,
+		{
+			headers: {
+				'content-type': 'application/json',
+				'User-Agent': 'Climate Conversations Proxy',
+				'authorization': `Bearer ${token}`,
+			},
+			method: 'GET',
+		}
+	);
+
+	const usersBirthdayAbsent = await usersNextBirthdayAbsentResponse.json();
+
+	if (usersBirthdayAbsent.data.length == 0) {
+		console.log('No users with absent next birthdays found');
+	}
+
+	// remove invalid birthdays
+	const usersBirthdayAbsentValidated = usersBirthdayAbsent.data.filter(
+		(user) => dayjs(user.private.dateOfBirth).isValid()
+	);
```

Comment:
If it's not valid then we should set it to a valid value (if dateOfBirth is present) or clear it if it isn't